packages/supabase/test/client_test.dart

```diff
@@ -31,6 +32,29 @@ void main() {
           client.storage.headers['X-Client-Info']!.split('/').first;
       expect(xClientHeaderBeforeSlash, 'supabase-dart');
     });
+
+    test('realtime URL is properly being set', () {
+      var realtimeWebsocketURL = Uri.parse(client.realtime.endPointURL);
+      expect(
+        realtimeWebsocketURL.queryParameters,
+        containsPair('apikey', supabaseKey),
+      );
+      expect(realtimeWebsocketURL.queryParameters['log_level'], isNull);
+
+      client = SupabaseClient(supabaseUrl, supabaseKey,
+          realtimeClientOptions:
+              RealtimeClientOptions(logLevel: RealtimeLogLevel.info));
```

Comment:
```suggestion
              RealtimeClientOptions(logLevel: RealtimeLogLevel.info),);
```
And reformat

Reply:
We can add [require_trailing_comma](https://dart.dev/tools/linter-rules/require_trailing_commas) lint rule if we want to enforce these. Otherwise, I don't think we need to be so sensitive about code format as long as it's passing the lints.

Reply:
But yeah, I am all in for it to add `require_trailing_comma` to our lint. We can probably introduce a few more rules to improve our code base.